sections/futures/TradingHistory/TradesHistoryTable.tsx

```diff
@@ -199,6 +200,13 @@ const HistoryContainer = styled.div<{ mobile?: boolean }>`
 		`}
 `;
 
+const TableMainHeader = styled.div`
+	font-size: 13px;
+	color: #ece8e3;
```

Comment:
i've added `color: ${(props) => props.theme.colors.selectedTheme.text.value};` instead `color: ${(props) => props.theme.colors.selectedTheme.text.title};` because it has white color.